### ISSUE TITLE
add support for “current products” meta target in Repositories

### ISSUE THREAD

**Issue by stanislav-brabec** [openSUSE/open-build-service#795]
Most project maintainers want to build their code for new openSUSE versions.

Targets change every year, but repositories are not updated. They keep building for obsolete openSUSE versions and missing the latest openSUSE.

It would be good to add set of meta targets, which will represent current products. Its change will auto-update targets in all projects that use these tags.

Example of the targets:
Supported openSUSE products
Latest openSUSE
Supported SLE products
Latest SLE
Latest Fedora
Latest Debian
Latest Centos
Supported Centos
…

**Comment by stanislav-brabec** [comment]
Is there any decision/progress?

I spent 30 minutes in October by adding of openSUSE_13.2 to all involved projects.

Now I spent more than one hour by removing of openSUSE_12.3 from all involved projects, checking xUbuntu web for unsupported xUbuntu versions and removing them as well. (Removing from most projects is not possible in the web interface, as there are linked projects, and I had to use "osc meta prj -f -e" for them.)

**Comment by adrianschroeter** [comment]
The removal step is usually done via auto-accepting requests. But I have not started that for 12.3 yet.

Still, I do not like the idea much, at least as long we do not have an answer to the question how to detect not anymore active projects.

**Comment by adrianschroeter** [comment]
I will run the removal request job today again JFYI.

**Comment by stanislav-brabec** [comment]
Yes, this is a drawback of my proposal: Abandoned projects using meta targets would get fresh repositories forever.

Maybe projects (or meta tags) could have a lifetime reset on repo changes or on explicit requests.

Well a something similar already exists: GPG signatures. Meta tags could expand to empty string for repositories with expired signature.
